### ISSUE TITLE
Would it be possible to use it with the Whisper API instead of letting it run locally. It's just not the same with these small models..



### ISSUE THREAD

**Issue by VincentLuders** [futo-org/voice-input#82]

**Comment by AndreasKueck** [comment]
[Dictate Keyboard (Whisper AI)](https://play.google.com/store/apps/details?id=net.devemperor.dictate&pli=1)

**Comment by tashijayla** [comment]
FUTO Voice Input will never support online speech-to-text. It is an offline speech-to-text app.

So you're barking up the wrong tree if you'd like FUTO voice input to introduce an internet-based speech-to-text API because it will never be done.

**Comment by VincentLuders** [comment]
Ok. Who said never?.

Do you have any alternatives?

**Comment by AndreasKueck** [comment]
> does the app that you are referring to have this feature?

I am not sure I understand what you mean, but I tend to answer: "No, it hasn't."

**Comment by AndreasKueck** [comment]
> Do you have any alternatives?

No, I haven't. [I use FUTO Voice Input and Dictate for different purposes.](https://gist.github.com/AndreasKueck/70d1a4e33843289954114990e71e6ab4?permalink_comment_id=5149108#gistcomment-5149108)
